src/Conversion/DecToRoman.php

```diff
@@ -9,14 +9,12 @@
 class DecToRoman
 {
 
-	private $symbolMap;
+	private $symbolMap = [['I', 'V'], ['X', 'L'], ['C', 'D'], ['M']];
 
 	public function __construct(array $symbolMap = [])
 	{
 		if ($symbolMap !== []) {
 			$this->symbolMap = $symbolMap;
-		} else {
-			$this->symbolMap = [['I', 'V'], ['X', 'L'], ['C', 'D'], ['M']];
```

Comment:
I'd really like to have properties initialized in constructors of the class. It offers more variability and keeps potential logic in one place (as like exactly in this case).